application/views/main.php

```diff
@@ -5,7 +5,7 @@
         $title = "";
         switch ($page) {
             case 'auth/login':
-                $title = 'Masuk';
+                $title = 'Masuk 1';
```

Comment:
nang ini typo ya, harusnya gini
```suggestion
                $title = 'Masuk';
```